includes/wpmenucart-settings.php

```diff
@@ -454,33 +452,36 @@ public function wpmenucart_admin_styles() {
 	 * Default settings.
 	 */
 	public function default_settings() {
-		$wcmenucart_options = get_option('wcmenucart');
-		$menu_slugs = array( '1' =>  isset($wcmenucart_options['menu_name_1']) ? $wcmenucart_options['menu_name_1']:'0' );
+		$option              = 'wcmenucart';
+		$wcmenucart_options  = get_option( $option, array() );
+		$menu_slugs          = array(
+			'1' => isset( $wcmenucart_options['menu_name_1'] ) ? $wcmenucart_options['menu_name_1'] : '0'
```

Comment:
WPCS - [For an array with elements in multiple lines, adding a comma at the end is recommended.](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#:~:text=Note%20the%20comma%20after%20the%20last%20array%20item%3A%20this%20is%20recommended%20because%20it%20makes%20it%20easier%20to%20change%20the%20order%20of%20the%20array%2C%20and%20makes%20for%20cleaner%20diffs%20when%20new%20items%20are%20added.)